dist/images/ovnkube.sh

```diff
@@ -672,6 +672,11 @@ ovs-server() {
     echo "Warning: Higher memory allocation by ovs-vswitchd is expected due to high number of n-handler-threads and n-revalidator-threads"
   fi
 
+  # Reduce stack size to 2M from default 8M as per below commit on Openvswitch
+  # https://github.com/openvswitch/ovs/commit/b82a90e266e1246fe2973db97c95df22558174ea
+  # added while troubleshooting on https://bugzilla.redhat.com/show_bug.cgi?id=1572797 
+  ulimit -s 2048
```

Comment:
I dont see this being applied in my KinD cluster within `ovs-node-*` pod:

```shell
$oc -n ovn-kubernetes rsh ovs-node-j6wc9
sh-5.2# ulimit -s
8192
```

Reply:
My bad, I should have set ulimit before starting ovs-vswitchd. I will modify and recommit.

 `ulimit -s` gives session specific output. I am verifying the fix by checking content of /proc/$(pidof ovs-vswitchd)/limits inside ovs-node POD.